test/Base_Test.t.sol

```diff
@@ -319,7 +264,101 @@ abstract contract Base_Test is Test {
         assertEq(succ, false);
     }
 
-    function test_revertsWhenNoProperFunctionSelectorUsed(bytes4 selector) public virtual {
+    function test_sendZeroAmount() public virtual hasSafetyChecks {
```

Comment:
Important to note: None of the contracts revert if `0` is sent. Is this desired behavior?